charts/thurloe/README.md

```diff
@@ -0,0 +1,5 @@
+Thurloe
```

Comment:
This isn't necessary for merging, but @gmalkov introduced us all to a tool called `helm-docs` for generating the README. It's awesome!

Copy [a README template](https://github.com/broadinstitute/terra-helm/blob/master/charts/workspacemanager/README.md.gotmpl) from another app. Then you can [annotate values](https://github.com/broadinstitute/terra-helm/blob/master/charts/workspacemanager/values.yaml) so they are automatically added to the README.